src/components/Tooltip/README.md

```diff
@@ -69,7 +69,7 @@ Use only when necessary to provide an explanation for an interface element.
 ```jsx
 <div style={{padding: '75px 0'}}>
   <Tooltip active content="This order has shipping labels.">
-    <Link>Order #1001</Link>
+    <TextStyle variation="strong">Order #1001</TextStyle>
```

Comment:
Should we revert this example change?

Reply:
I think it is actually useful to have one example that isn't a link here. The link never showed this being an issue because it had its own focus styles.